integration/delete_before_lockdown.rs

```diff
@@ -0,0 +1,22 @@
+
+use birdcage::{Birdcage, Exception, Sandbox};
+use tempfile::NamedTempFile;
+
+use crate::TestSetup;
+
+pub fn setup() -> TestSetup {
+    // Create temporary file.
+    let tempfile = NamedTempFile::new().unwrap();
+
+    // Setup sandbox exceptions.
+    let mut sandbox = Birdcage::new();
+    sandbox.add_exception(Exception::Read(tempfile.path().into())).unwrap();
+
+    drop(tempfile);
```

Comment:
It might be slightly better to use `.close()` since it will return `Err(_)` if there is an error deleting the file (which would invalidate this test).

```suggestion
    tempfile.close().unwrap();
```